random_menu.rb

```diff
@@ -0,0 +1,88 @@
+# Create a random menu generator that can be played from the Terminal.
+#
+# Your generator should pull one item from each array you made in the baseline requirements to create a "menu item".
+#
+# When the program runs, it should create and show a list of ten numbered menu items.
+#
+# 1. hot pan-fried dumplings
+# 2. soft steamed clams
+# 3. ...
+# ...
+# 10. creamy taco cake
+# Test & Verify
+#
+# Before you submit your work it's important to test your program and ensure it's working properly.
+#
+# Among your tests ensure that:
+#
+# The menu items are selected randomly.
+# There are 10 menu items numbered 1 - 10 (not starting at zero)
+# Each item should pull one word from each of the 3 arrays.
+
+
+menu_items = 10
+
+food_adjectives = [
+  "Crunchy",
+  "Soft",
```

Comment:
I like that you put these on separate lines. It takes up more vertical space but is very readable.